### DISCUSSION TITLE
I can not change the the mqtt topic?

### DISCUSSION THREAD

**Discussion by florian-reus** [arendst/Tasmota#14623] [Q&A] [answered]
hi,

it should be simple, but is not working:
_16:24:57.423 CMD: topic display
16:24:57.430 MQT: stat/tasmota_10D6D7/RESULT = {"Topic":"tasmota_%06X"}_

Also in GUI I can "save" the new topic, it's restarting, but it's not changed.
It happend after I deleted the device from HA (had several strange sensors), and rebooted through the web interface.

Any ideas? 

Program Version	10.0.0(TasmoCompiler-esp82664M)
Build Date & Time	2021-11-19T19:23:38
Core/SDK Version	2_7_4_9/2.2.2-dev(38a443e)

**Comment by sfromis** [comment, marked as answer]
Topic and MqttClient can't be the same. This restriction has been lifted if you upgrade to newest dev version.

**Reply by florian-reus** [reply]
thanks!
Helped.
